ignite/cmd/scaffold_message.go

```diff
@@ -13,7 +13,7 @@ const flagSigner = "signer"
 // NewScaffoldMessage returns the command to scaffold messages.
 func NewScaffoldMessage() *cobra.Command {
 	c := &cobra.Command{
-		Use:   "message [name] [field1] [field2] ...",
+		Use:   "message [name] [field1] [field2:type2] ...",
```

Comment:
```suggestion
		Use:   "message [name] [field1:type1] [field2:type2] ...",
```

@fadeev I believe types were not present before this PR, I think they should be specified because they are part of the command line "API", what do you think?

Reply:
Tricky, because types are optional, let's add types.

Reply:
Maybe even make it more readable:

```
message [name] [field:type] [field:type]
```